### ISSUE TITLE
Form settings page inconsistencies

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#1713]
# Issue/opportunity

`Settings` does not bold to show it was selected on left-hand form menu (other elements do: `Edit` / `Test` / `Publish`

This page also seems to be missing an H1 to indicate the page title?

# Screenshots

<img width="1495" alt="Screenshot 2023-03-08 at 12 59 32 PM" src="https://user-images.githubusercontent.com/38330843/223793529-e4c3b5b0-6091-45f0-b097-cbf3c49ee86d.png">

<img width="1463" alt="Screenshot 2023-03-08 at 12 59 42 PM" src="https://user-images.githubusercontent.com/38330843/223793502-2736f62a-db48-4acb-918d-de70547cce82.png">

**Comment by timarney** [comment]
Please confirm this has been fixed on staging https://forms-staging.cdssandbox.xyz/form-builder/settings

**Comment by timarney** [comment]
As for the heading that should be brought back to a design discussion as it varies per page

A new navigation has also been added since this issue came up

Settings Figma:
https://www.figma.com/file/6QAmUEq6d8tyXzWDGKO78s/v2.-Form-builder?node-id=2307-50134&t=H2U51zav4WIIvNLC-0

Current
<img width="870" alt="Screen Shot 2023-03-27 at 11 08 15 AM" src="https://user-images.githubusercontent.com/62242/227982460-afa4b68b-f30a-4fee-88ca-22125473711f.png">

The classification dropdown is in 

https://github.com/cds-snc/platform-forms-client/pull/1780

**Comment by anikbrazeau** [comment]
I believe this is resolved with bolding now present and new nav format.
